transport/tchannel/handler.go

```diff
@@ -201,25 +209,30 @@ func (h handler) callHandler(ctx context.Context, call inboundCall, responseWrit
 
 	body, err := call.Arg3Reader()
 	if err != nil {
+		UpdateSpanWithErr(span, err, yarpcerrors.FromError(err).Code())
```

Comment:
I though UpdateSpanWithErr returns err and so it can be used as `return UpdateSpanWithErr(...`?